src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriors.java

```diff
@@ -23,19 +24,19 @@
 import java.util.*;
 
 /**
- * Calculate genotype posterior probabilities given panel data
+ * Calculate genotype posterior probabilities given family genotypes or known population genotypes
  *
  * <p>
- * Given a VCF with genotype likelihoods from the HaplotypeCaller, UnifiedGenotyper, or another source which provides
- * <b>unbiased</b> genotype likelihoods, calculate the posterior genotype state and probability given allele frequency
- * information from both the samples themselves and input VCFs describing allele frequencies in related populations.</p>
+ * Given a VCF with genotype likelihoods from HaplotypeCaller, UnifiedGenotyper, or another source which provides
+ * <b>unbiased</b> genotype likelihoods, calculates the posterior genotype probability given genotype likelihoods
+ * from both the samples themselves and input VCFs describing allele frequencies in related populations.</p>
  *
  * <p>The AF field will not be used in this calculation as it does not provide a way to estimate the confidence interval
  * or uncertainty around the allele frequency, while AN provides this necessary information. This uncertainty is
  * modeled by a Dirichlet distribution: that is, the frequency is known up to a Dirichlet distribution with
  * parameters AC1+q,AC2+q,...,(AN-AC1-AC2-...)+q, where "q" is the global frequency prior (typically q << 1). The
  * genotype priors applied then follow a Dirichlet-Multinomial distribution, where 2 alleles per sample are drawn
- * independently. This assumption of independent draws is the assumption Hardy-Weinberg Equilibrium. Thus, HWE is
+ * independently. This assumption of independent draws is assuming Hardy-Weinberg Equilibrium. Thus, HWE is
```

Comment:
I would say "This assumption of independent draws follows from the assumption of Hardy-Weinberg equilibrium."